packages/shared/sierra-client/src/HttpSierraClient.ts

```diff
@@ -110,6 +110,45 @@ export default class HttpSierraClient implements SierraClient {
     });
   }
 
+  async getDeletedRecordNumbers({
+    start,
+    end,
+  }: {
+    start?: Date;
+    end?: Date;
+  } = {}): Promise<APIResponse<number[]>> {
+    try {
+      const instance = await this.getInstance();
+      const startDateString = start?.toISOString().slice(0, 10);
+      const endDateString = (end ?? new Date()).toISOString().slice(0, 10);
+
+      const deletedDate = startDateString
+        ? `[${startDateString}, ${endDateString}]`
+        : undefined;
+
+      const response = await instance.get<{ entries: Array<{ id: number }> }>(
+        '/patrons',
+        {
+          params: {
+            deleted: true,
+            deletedDate,
+          },
+          validateStatus: (status) => status === 200,
+        }
+      );
```

Comment:
What if there are more deletions than fit in a single page? Will this correctly paginate over them, or does it just get the first page?

Reply:
I wonder how many times I have written, and how many times more I will write, a wrapper for a paginated API